commcare_connect/static/js/vendors.js

```diff
@@ -1,7 +1,23 @@
 import '@popperjs/core';
-import 'bootstrap';
+import * as bootstrap from 'bootstrap';
 import 'htmx.org';
 import Alpine from 'alpinejs';
 
 window.Alpine = Alpine;
 Alpine.start();
+
+function refreshTooltips() {
```

Comment:
what does this function do?

Reply:
This function is used for loading the tooltips on page load or table load in this case.

Reply:
why is it in `vendors/js`? this file seems to be around importing third party libraries

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/252/commits/b5f33d9d99fea130444bce51fe5e71d09340100b